### DISCUSSION TITLE
UBLOCK??

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1369] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
how do I use UBLOCK to disable extensions.??

**Comment by S-PScripts** [comment]
https://github.com/s-pscripts/chromebook-utilities/blob/main/Exploits/GoGuardian%20uBlock
https://github.com/s-pscripts/chromebook-utilities/blob/main/Exploits/Run%20Code%20(with%20uBlock)
https://github.com/s-pscripts/chromebook-utilities/blob/main/Exploits/Run%20Code%202%20(uRun%20for%20uBlock)
https://github.com/s-pscripts/chromebook-utilities/blob/main/Exploits/Securly%20uBlock
